server/lobbyconnection.py

```diff
@@ -1038,7 +1038,9 @@ async def launch_game(
             # options are. Currently, options for ladder are hardcoded into the
             # client.
             "name": game.name,
+            # Depricated: init_mode can be inferred from game_type
             "init_mode": game.init_mode.value,
+            "game_type": game.game_type.value,
```

Comment:
These should be strings. That’s part of the reason for switching.

Reply:
both of them ?

Reply:
No just GameType. Init mode needs to stay as it is since it’s being used currently, changing it would break the client

Reply:
is it better to have the enum members be strings or are to/from_string methods preferred ?

Reply:
I think making them be strings is fine. I’m honestly not sure anymore why we have the to/from methods. I don’t think there was any particular reason though, just didn’t think of doing it the other way.